### ISSUE TITLE
Preference for Siddhanta font for Devanagari

### ISSUE THREAD

**Issue by funderburkjim** [sanskrit-lexicon/Cologne#163]
A user recently submitted in the Correction Form a complaint about how the 'kt' in 'ukta' is displayed
in Devanagari.  The comment was made regarding the MD dictionary, and probably the Basic display.

I'm putting his comment, and my response here and labeling as enhancement.  We should aim to
make the display of Devanagari similar (and using Siddhanta font) for all the displays of all the dictionaries.
 
```
Case 23748: 06/24/2017 dict=MD, L=4266, hw=ukta, user=torbium
old = उक्त
new = the workd ukta. Different ligatura for kta seems to be needed.
comment = the workd ukta. The kta ligatura seems strange. On the scan of book page provided the ligatur is correntc one.
```

Here's an explanation of what's going on with Devanagari ligatures.  The details of a given ligature depend on exactly which

font is being used by the browser; and this in turn depends on exactly how the font is specified in the html page which the

browser is rendering.  

In the display at url http://www.sanskrit-lexicon.uni-koeln.de/scans/MDScan/2014/web/webtc/indexcaller.php,  Devanagari

is being rendered with the default unicode font (i.e., it depends on the Computer operating system).  For instance, I am using

Windows 10, and the default font for Devanagari is Nirmala UI  (I discovered this by using the developer tools  of the Chrome browser).

And in this font 'ukta' in Devanagari renders as .
![image](https://user-images.githubusercontent.com/6393033/28190287-2fad5ebc-67f8-11e7-830b-9956a3fe53c4.png)

There is another display at Cologne at url http://www.sanskrit-lexicon.uni-koeln.de/scans/awork/apidev/sample/list-0.2.html.

This display specifically sets the font for Devanagari to Siddhanta .  In Siddhanta font, ukta looks like 
![image](https://user-images.githubusercontent.com/6393033/28190303-3e86ae16-67f8-11e7-9567-dfcbaabaf511.png)

Other people have expressed a preference for the Siddhanta font, and we may at some time change all the displays to use Siddhanta

font.  But for now, you'll need to use the list-0.2.html display to get Siddhanta for MD.

**Comment by gasyoun** [comment]
> you'll need to use the list-0.2.html display to get Siddhanta for MD

Exactly. I'm for Siddhanta, but that's only UI. We stall have not implemented Simple Search - that will change the whole way we search. That's what's important.

**Comment by drdhaval2785** [comment]
Siddhanta is implemented in Advanced (which I use most often) and maybe in all displays.
There is one query.
When I search for 'Davala', I see the following.
```
धवल [Printed book page 0433-b]
धवल dhav + ala (cf. 2. dhāv), adj.,
f. lā. 1. Dazzling white, Kathās. 25,
15. — Comp. Ati-, adj. very white, Daśak.
in Chr. 199, 6. [ID=6979]
```

Rest is in Siddhanta, but the following part is in FreeSerif
```
धवल [Printed book page 0433-b]
```

@funderburkjim, is it intentional or accidental?

**Comment by funderburkjim** [comment]
This is accidental (AKA, a bug).

**Comment by drdhaval2785** [comment]
Small change, Jim? Request to remove the bug. If you intend to use Adhishila, then also this would require attention.

**Comment by Andhrabharati** [comment]
If I may say this,  why not use the same font whichever is used in the body area at the header portion as well (but just in **bold** letters here)?

And use the same font groups in all kinds of search pages.
